docscan/session/create/filter/document_restrictions_filter.go

```diff
@@ -4,8 +4,9 @@ import "encoding/json"
 
 // RequestedDocumentRestrictionsFilter filters for a required document, allowing specification of restrictive parameters
 type RequestedDocumentRestrictionsFilter struct {
-	inclusion string
-	documents []*RequestedDocumentRestriction
+	inclusion             string
+	documents             []*RequestedDocumentRestriction
+	allowExpiredDocuments *bool
```

Comment:
Do we always use pointers here? Why?

Reply:
I guess I understand, if the defaults are set on the backend then it can make sense.

Reply:
Yes you are right